app/assets/stylesheets/govuk_publishing_components/components/_step-by-step-nav-header.scss

```diff
@@ -3,8 +3,8 @@
   position: relative;
   padding: 10px;
   background: govuk-colour("light-grey", $legacy: "grey-4");
-  border-top: solid 1px govuk-colour("mid-grey", $legacy: "grey-2");
-  border-bottom: solid 1px govuk-colour("mid-grey", $legacy: "grey-2");
+  border-bottom: solid 1px govuk-colour("blue");
```

Comment:
Do we know what was the reasoning for changing the border colour? We'll still have to preserve the `$legacy` attribute since the compatibility mode is still on in some applications.

Reply:
Not sure what the reason is for the blue line but here is the trello - https://trello.com/c/IOjnO55X/196-update-superbreadcrumb-component-ready-for-coronavirus-tagged-content

I've added in the legacy colour

Reply:
Thank you! Don't have access to the board, unfortunately.

Reply:
Just to clarify, you only _need_ to provide a `$legacy` attribute if the colour with that name does not exist in the legacy palette – which is the case with `mid-grey` (the legacy palette has `grey-1` through `grey-4` instead).

As there is a colour called `blue` in both the new and legacy palettes, it'll just use the blue from whichever respective palette, without needing to pass `$legacy`.

Reply:
ah, thank you for pointing this out @36degrees. my bad, @DilwoarH 🤦 , `blue` is still `blue`, I was thinking we'll go with `mid-gray`.

Reply:
Thanks for confirming this @36degrees

@alex-ju - most of the information about the context is in the PR description. Here is a copy of the design that I was given:
<img width="375" alt="Corona_content_nav_mob-1" src="https://user-images.githubusercontent.com/4599889/80604475-b00aa000-8a29-11ea-8214-89c537b91641.png">

Reply:
Well, assuming this was well thought by a designer, I'm happy to get it in.